typings/nerdamer-defs-1.1.4.d.ts

```diff
@@ -1,48 +1,53 @@
 // Code - https://github.com/jiggzson/nerdamer/blob/master/nerdamer.core.js
 
-interface MathExpr {
-    // Returns all variables in the given MathExpr.
+namespace nerdamer {
+  interface Expression {
+    // Returns all variables in the given Expression.
     variables: () => string[];
 
-    // Returns the text representation of the MathExpr.
+    // Returns the text representation of the Expression.
     text: (option: string, n: string) => string;
 
-    // Evaluates the MathExpr into the simplest form.
-    evaluate: () => MathExpr;
+    // Evaluates the Expression into the simplest form.
+    evaluate: () => Expression;
 
     // Returns the numerator of the polynomial.
-    numerator: () => MathExpr;
+    numerator: () => Expression;
 
     // Returns the denominator of the polynomial.
-    denominator: () => MathExpr;
+    denominator: () => Expression;
 
-    // Adds two MathExprs.
-    add: (x: string) => MathExpr;
+    // Adds two Expressions.
+    add: (x: string) => Expression;
 
-    // Subtracts two MathExprs.
-    subtract: (x: string) => MathExpr;
+    // Subtracts two Expressions.
+    subtract: (x: string) => Expression;
 
-    // Multiplies two MathExprs.
-    multiply: (x: string) => MathExpr;
-    
-    // Divides two MathExprs.
-    divides: (x: string) => MathExpr;
+    // Multiplies two Expressions.
+    multiply: (x: string) => Expression;
 
-    // Raises one MathExpr to another.
-    pow: (x: string) => MathExpr;
+    // Divides two Expressions.
+    divide: (x: string) => Expression;
 
-    // Checks for equality of two MathExprs.
+    // Raises one Expression to another.
+    pow: (x: string) => Expression;
+
+    // Checks for equality of two Expressions.
     eq: (x: string) => boolean;
 
-    // Checks if one MathExpr is less than other.
+    // Checks if one Expression is less than other.
     lt: (x: string) => boolean;
 
-    // Checks if one MathExpr is greater than other.
+    // Checks if one Expression is greater than other.
     gt: (x: string) => boolean;
 
-    // Checks if one MathExpr is less than, or equal to, other.
+    // Checks if one Expression is less than, or equal to, other.
     lte: (x: string) => boolean;
 
-    // Checks if one MathExpr is greater than, or equal to, other.
+    // Checks if one Expression is greater than, or equal to, other.
     gte: (x: string) => boolean;
-}
+
+    // Expands the Expression.
+    expand: () => Expression;
+  }
+}
```

Comment:
Add a new line here.

 Also I've noticed the lint checks don't work in the `typings` directory. Is it expected?

/cc @Hudda @kevinlee12

Reply:
Done.

Reply:
> Also I've noticed the lint checks don't work in the typings directory. Is it expected?

Yep lint checks don't work here, `typings` directory is excluded from lint checks